pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -248,6 +252,10 @@ type TracepointSpec struct {
 	// +kubebuilder:validation:Optional
 	// Selectors to apply before producing trace output. Selectors are ORed.
 	Selectors []KProbeSelector `json:"selectors,omitempty"`
+	// +kubebuilder:validation:Optional
+	// Tags to categorize the event, will be include in the event output.
+	// Maximum of 16 Tags are supported.
```

Comment:
Nothing is enforcing this maxiumum at least as far as I can see here. So why bother with a max at all? It looks like a configured CRD value.

Reply:
So the enforcing is happening when parsing the crd, we don't have limits for other field and I'm not sure let me think

Reply:
My thought is if its just a slice does it matter what the max is?